Sources/SwiftDocC/Infrastructure/Symbol Graph/SymbolGraphLoader.swift

```diff
@@ -146,6 +146,36 @@ struct SymbolGraphLoader {
         
         self.symbolGraphs = loadedGraphs.mapValues(\.graph)
         (self.unifiedGraphs, self.graphLocations) = graphLoader.finishLoading()
+        
+        unifiedGraphs.forEach { (_, unifiedGraph) in
+            var unifiedGraph = unifiedGraph
+            var defaultUnconditionallyUnavailablePlatforms = [PlatformName]()
+            var defaultAvailableAvailabilities = [DefaultAvailability.ModuleAvailability]()
```

Comment:
I find both of these variable names hard to read but I'm struggling to come up with better names without making them equally wordy. 

I think the best I've got is `platformsToMarkUnavailableAsFallback` (focusing on what it's used for) and `defaultIntroducedVersions` (trying to describe the specific availability information that this represents).

I used different words ("fallback" and "default") to mean the same thing on those variable names. It may be clearer with the same word in both variables? What do you think of those alternatives?

Reply:
I prefer using `default` for both because they refer to information from the default availability section in the Info.plist. Using `fallback` could mistakenly associate it with the term fallback platforms, which we already use for referring to fallback platforms for availability.

I agree that the current names are verbose and very similar, so I suggest renaming them to `defaultUnavailablePlatforms` and `defaultAvailableInformation`.

Reply:
https://github.com/apple/swift-docc/pull/807/commits/c55abc329895424275b96a3c47afef3f8c5bb3f9